### ISSUE TITLE
Middleware does not work with HAPI 17

### ISSUE THREAD

**Issue by mkochendorfer** [APIs-guru/graphql-voyager#65]
HAPI 17 changed its plugin registration format, and thus the middleware here needs to be updated to match.

https://github.com/prismagraphql/graphql-playground/commit/a4ddbd62332b73bb67773ddd58dfecb76f1ef28d#diff-63ec1e6c31c625b1529c90ef9da62e18 has a good reference of the changes need in the middleware to get this working again in HAPI 17

**Comment by IvanGoncharov** [comment]
@mkochendorfer Thanks for reporting 👍 
PR is welcome 🎉

**Comment by mkochendorfer** [comment]
@IvanGoncharov What branch should I PR against since this will not be backwards compatible with old HAPI versions?

**Comment by IvanGoncharov** [comment]
@mkochendorfer Middleware files are pretty small so just create a new one `hapiModern` or something similar.
Maybe rename old one to `hapiLegacy`?

**Comment by mkochendorfer** [comment]
https://github.com/APIs-guru/graphql-voyager/pull/66

**Comment by mkochendorfer** [comment]
Here is a temporary workaround for anyone who needs HAPI 17 before support is added to the middleware here:

```
// Hack to convert graphql-voyager/middleware plugin to HAPI 17.
// Can be removed once https://github.com/APIs-guru/graphql-voyager/issues/65 is fixed.
import { hapi as graphqlVoyager, renderVoyagerPage } from 'graphql-voyager/middleware';

const graphqlVoyagerUpdated = {
  pkg: graphqlVoyager.attributes.pkg,
  register: (server, options) => {
    const {
      path,
      route,
      ...voyagerOptions
    } = options;

    server.route({
      method: 'GET',
      path,
      config: route,
      handler: (_request, h) =>
        h.response(renderVoyagerPage(voyagerOptions)),
    });
  },
};
```

**Comment by IvanGoncharov** [comment]
Fixed in #66 
Released as 1.0.0-rc.16 📦
